### ISSUE TITLE
Toast: close button missing text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#870]
### Expected behavior

Close button has text to communicate its function. 
### Actual behavior

Close button has "x" character as its text label.
